pkg/log/pipeline_reader.go

```diff
@@ -120,45 +121,47 @@ func (r *Reader) readAvailablePipelineLogs(pr *v1beta1.PipelineRun) (<-chan Log,
 // only if run status is unknown, open a watch channel on run
 // and keep checking the status until it changes to true|false
 // or the reach timeout
-func (r *Reader) waitUntilAvailable(timeout time.Duration) error {
+func (r *Reader) waitUntilAvailable(timeout time.Duration) (*v1beta1.PipelineRun, error) {
```

Comment:
if the first return value is getting ignored, we can avoid that then, no need to return